com._2006Scape.Launcher.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com._2006Scape.Launcher</id>
+  <launchable type="desktop-id">com._2006Scape.Launcher.desktop</launchable>
+  <name>2006Scape</name>
+  <developer_name>2006Scape</developer_name>
+  <summary>Launcher for 2006Scape</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0-or-later</project_license>
+  <url type="homepage">https://2006Scape.org/</url>
```

Comment:
this is .org but the app-id uses .com. And they are not the same. What gives?

Reply:
using com. was a mistake